src/main/java/com/mael/ttt/ui/UserInterface.java

```diff
@@ -105,4 +108,12 @@ public boolean replay() {
         print(REPLAY);
         return console.read().equals("y");
     }
+
+    public List<String> getBoardIndexesAsStrings(int size) {
```

Comment:
Since the method just takes a size and returns a list with the numbers from `1` to `size` as Strings, I wonder if I should just rename the method to avoid confusion. Something like `getNumbersFromOneToNAsStrings(int n)`